opencsp/app/sofast/lib/CalibrateSofastFixedDots.py

```diff
@@ -140,7 +140,7 @@ def __init__(
         self._dot_image_points_indices: Vxy
         self._dot_image_points_indices_x: ndarray
         self._dot_image_points_indices_y: ndarray
-        self._dot_points_xyz_mat = np.ndarray((x_max - x_min + 1, y_max - y_min + 1, 3)) * np.nan
+        self._dot_points_xyz_mat = np.ndarray((y_max - y_min + 1, x_max - x_min + 1, 3)) * np.nan
```

Comment:
For my own edification, what does multiplying by np.nan accomplish?

Reply:
Those matrices get filled in with data as the algorithm finds dots. Since some values may actually be zero, we need some other value to define "no data." NaN seemed appropriate.